backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/tokens/BpkColor.kt

```diff
@@ -14,256 +14,233 @@
 
 package net.skyscanner.backpack.compose.tokens
 
-import androidx.compose.runtime.Stable
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
 import androidx.compose.ui.graphics.Color
 
 public object BpkColor {
-  @Stable
-  public val BackgroundAlternativeDark: Color = Color(0xFF000000)
+  public val Abisko: Color = Color(0xFF5A489B)
 
-  @Stable
-  public val BackgroundAlternativeLight: Color = Color(0xFFF1F2F8)
+  public val Background: Color
+    @Composable
+    get() = if (MaterialTheme.colors.isLight) White else Black
 
-  @Stable
-  public val BackgroundAlternativeSecondaryDark: Color = Color(0xFF1D1B20)
+  public val BackgroundAlternative: Color
+    @Composable
+    get() = if (MaterialTheme.colors.isLight) SkyGrayTint07 else Black
```

Comment:
I wonder if we should keep the semantic colours in a separate file to make it clearer what's semantic? Or actually define these as the theme colours instead?

Reply:
> Semantic (context) colours (e.g. background/onBackground) may exist there, as it does in Material Design, but just colours values seems to be irrelevant to be put in there.
I'm not sure I understand? I think the dynamic colours are semantic and they reference other static colours? That also seems to match with material design where the theme defines the semantic colours in light + dark mode and uses the static colours defined elsewhere,

> It's also expected that Background (as all non-named such as "Abisko" colors) are dynamic by their nature – do it should make sense in non IDE environments (during code review).
Colours like `backgroundDark` are not dynamic, though - maybe the dark/light part does indicate that, but as there's actually a distinction at parsing level and we now have the ability to split them up it might be worthwhile to avoid people using non-dynamic colours by accident.

Alternatively: what would be your idea for the theme colours? would they be a manual list of some of these colours (and potentially some more)?